packages/integration-tests/helpers/childChainHelper.js

```diff
@@ -112,23 +115,24 @@ async function createTx (childChain, from, to, amount, currency, fromPrivateKey,
   if (!utxosToSpend || utxosToSpend.length === 0) {
     throw new Error(`Not enough funds in ${from} to cover ${amount} ${currency}`)
   }
-
   // Construct the tx body
   const txBody = {
     inputs: utxosToSpend,
     outputs: [{
-      owner: to,
+      outputType: 1,
+      outputGuard: to,
       currency,
       amount
     }]
   }
 
-  const bnAmount = numberToBN(utxosToSpend[0].amount)
-  if (bnAmount.gt(numberToBN(amount))) {
+  const utxoAmount = utxosToSpend[0].amount
```

Comment:
I think the BigNumbers were necessary here. Otherwise it could calculate the wrong change amount when using large numbers (very common when dealing with wei)